### PR TITLE
[Vectorize] Vectorize metadata filtering edits

### PR DESCRIPTION
**Pull request by vy-ton**
- add breadcrumbs to metadata filtering docs
- clarify namespace vs metadata filtering
- add operator table for better structure

### CODE REVIEW

File: content/vectorize/learning/metadata-filtering.md [hunk 1]
```diff
@@ -14,11 +14,22 @@ By using metadata filtering to limit the scope of a query, you can filter by spe
...
+| Operator | Description |
+| -------- | ----------- |
+| `$eq`    | Equals      |
+| `$ne`    | Not equeals |
+
 - `filter` must be non-empty object whose compact JSON representation must be less than 2048 bytes.
 - `filter` object keys cannot be empty, contain `" | .` (dot is reserved for nesting), start with `$`, or be longer than 512 characters.
 - `filter` object non-nested values can be `string`, `number`, `boolean`, or `null` values.
 
+### Namespace vs. metadata filtering
```
Comment (vy-ton) [inline content/vectorize/learning/metadata-filtering.md]: @netgusto @ndisidore anything else to callout here?

Would saying namespace=logical partition vs metadata=arbitrary tags help?

File: content/vectorize/learning/metadata-filtering.md [hunk 2]
```diff
@@ -14,11 +14,22 @@ By using metadata filtering to limit the scope of a query, you can filter by spe
 
 Optional `filter` property on `query()` method specifies metadata filter:
 
-- Operators: `$eq` (equals), `$ne` (not equals).
+| Operator | Description |
+| -------- | ----------- |
+| `$eq`    | Equals      |
+| `$ne`    | Not equeals |
```
Comment (Maddy-Cloudflare) [inline content/vectorize/learning/metadata-filtering.md]:
```suggestion
| `$ne`    | Not equals |
```

File: content/vectorize/learning/metadata-filtering.md (hunk 1 above)
Comment (Maddy-Cloudflare) [inline content/vectorize/learning/metadata-filtering.md]:
```suggestion
### Namespace versus metadata filtering
```

File: content/vectorize/learning/metadata-filtering.md [hunk 3]
```diff
@@ -14,11 +14,22 @@ By using metadata filtering to limit the scope of a query, you can filter by spe
...
+
 - `filter` must be non-empty object whose compact JSON representation must be less than 2048 bytes.
 - `filter` object keys cannot be empty, contain `" | .` (dot is reserved for nesting), start with `$`, or be longer than 512 characters.
 - `filter` object non-nested values can be `string`, `number`, `boolean`, or `null` values.
 
+### Namespace vs. metadata filtering
+
+Both [namespaces](/vectorize/learning/insert-vectors/#namespaces) and metadata filtering narrow the vector search space for a query.  When evaluating both filter types, considerations include:
```
Comment (Maddy-Cloudflare) [inline content/vectorize/learning/metadata-filtering.md]:
```suggestion
Both [namespaces](/vectorize/learning/insert-vectors/#namespaces) and metadata filtering narrow the vector search space for a query.  Consider the following when evaluating both filter types:
```

File: content/vectorize/learning/metadata-filtering.md [hunk 4]
```diff
@@ -14,11 +14,22 @@ By using metadata filtering to limit the scope of a query, you can filter by spe
...
+
+- Namespace filter is applied first then metadata filter(s).
```
Comment (Maddy-Cloudflare) [inline content/vectorize/learning/metadata-filtering.md]:
```suggestion
- A namespace filter is applied before metadata filter(s). 
```

File: content/vectorize/learning/metadata-filtering.md [hunk 5]
```diff
@@ -14,11 +14,22 @@ By using metadata filtering to limit the scope of a query, you can filter by spe
...
 - `filter` must be non-empty object whose compact JSON representation must be less than 2048 bytes.
 - `filter` object keys cannot be empty, contain `" | .` (dot is reserved for nesting), start with `$`, or be longer than 512 characters.
 - `filter` object non-nested values can be `string`, `number`, `boolean`, or `null` values.
 
+### Namespace vs. metadata filtering
+
+Both [namespaces](/vectorize/learning/insert-vectors/#namespaces) and metadata filtering narrow the vector search space for a query.  When evaluating both filter types, considerations include:
+
+- Namespace filter is applied first then metadata filter(s). 
+- A vector can only be part of a single namespace with the documented [limits](/vectorize/platform/limits/). Whereas, vector metadata can contain multiple key-value pairs up to [metadata per vector limits](/vectorize/platform/limits/). Metadata values support different types (`string`, `boolean`, etc) so offer more flexibility.
```
Comment (Maddy-Cloudflare) [inline content/vectorize/learning/metadata-filtering.md]:
```suggestion
- A vector can only be part of a single namespace with the documented [limits](/vectorize/platform/limits/). Vector metadata can contain multiple key-value pairs up to [metadata per vector limits](/vectorize/platform/limits/). Metadata values support different types (`string`, `boolean`, and others), therefore offering more flexibility.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Comment by ndisidore** [comment]
LGTM

**Pull request merged**